[2021-05-25 21:29:39,557] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: covid19_cases_Jordan.Get_COVID19_Cases_For_Jordan_2020ToNow 2021-05-25T21:21:21.156058+00:00 [queued]>
[2021-05-25 21:29:39,704] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: covid19_cases_Jordan.Get_COVID19_Cases_For_Jordan_2020ToNow 2021-05-25T21:21:21.156058+00:00 [queued]>
[2021-05-25 21:29:39,710] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-05-25 21:29:39,711] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2021-05-25 21:29:39,712] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-05-25 21:29:39,908] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): Get_COVID19_Cases_For_Jordan_2020ToNow> on 2021-05-25T21:21:21.156058+00:00
[2021-05-25 21:29:39,951] {standard_task_runner.py:52} INFO - Started process 361 to run task
[2021-05-25 21:29:39,975] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'covid19_cases_Jordan', 'Get_COVID19_Cases_For_Jordan_2020ToNow', '2021-05-25T21:21:21.156058+00:00', '--job-id', '129', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/workflow.py', '--cfg-path', '/tmp/tmpupi4epjh', '--error-file', '/tmp/tmpjcdgjmoe']
[2021-05-25 21:29:40,026] {standard_task_runner.py:77} INFO - Job 129: Subtask Get_COVID19_Cases_For_Jordan_2020ToNow
[2021-05-25 21:29:40,844] {logging_mixin.py:104} INFO - Running <TaskInstance: covid19_cases_Jordan.Get_COVID19_Cases_For_Jordan_2020ToNow 2021-05-25T21:21:21.156058+00:00 [running]> on host 4e0af5d83701
[2021-05-25 21:29:41,842] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=waed
AIRFLOW_CTX_DAG_ID=covid19_cases_Jordan
AIRFLOW_CTX_TASK_ID=Get_COVID19_Cases_For_Jordan_2020ToNow
AIRFLOW_CTX_EXECUTION_DATE=2021-05-25T21:21:21.156058+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-05-25T21:21:21.156058+00:00
[2021-05-25 21:38:08,887] {local_task_job.py:77} ERROR - Received SIGTERM. Terminating subprocesses
[2021-05-25 21:38:08,950] {process_utils.py:100} INFO - Sending Signals.SIGTERM to GPID 361
[2021-05-25 21:38:08,958] {taskinstance.py:1239} ERROR - Received SIGTERM. Terminating subprocesses.
